src/parser/core/Combatant.ts

```diff
@@ -484,7 +484,7 @@ class Combatant extends Entity {
    * This id is the same as the spell ID on Wowhead.
    */
   hasLegendary(legendary: LegendarySpell) {
-    if (!this.scannedForLegendaries && this.legendaries.size === 0) {
+    if (!this.scannedForLegendaries && this.legendaries.size < 1) {
```

Comment:
What issue is resolved with this change?

Reply:
When the toon had more than one legendary, it was not picking up one of them.

Reply:
This does nothing, don't merge with this change in.
 < 1 is the same as === 0 except it would be true for negative size (impossible).

Reply:
@Pewtro @djanse reverted.